catalog-info.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: autodev-action
```

A description would be nice
```suggestion
  name: autodev-action
  description: This GitHub action merges labeled PRs into the dev branch.
```